### PR TITLE
i18n: auto detect locale from browser

### PR DESCRIPTION
**Pull request by Equim-chan**
How do you like the idea of auto detecting the default locale from the browser?

The implementation is a little complex and breaking though.

### CODE REVIEW

**Comment by hacdias** [comment]
Isn't that going to replace the user's settings? This is a good idea if the user doesn't set its own language.

**Comment by Equim-chan** [comment]
It should work that way, as `value.locale` is before `navigator.language`.
https://github.com/Equim-chan/filemanager/blob/0271b914a422dd6dbf7cbc843a05f0f45ec585ca/assets/src/store/mutations.js#L30

I also tested. After the user set his own language, the auto detection will be overrided.

The problem is the code isn't so DRY. The same switch case repeated twice in the code.

~~`switch (true)` style comes from go~~

File: assets/src/i18n/index.js [hunk 1]
```diff
@@ -9,8 +9,37 @@ import zhTW from './zh-tw.yaml'
...
+    locale = 'zh-tw'
+    break
+  case /^zh.*/i.test(locale):
+    locale = 'zh-cn'
+    break
+  default:
+    locale = 'en'
+}
+
+console.log(locale)
```
Comment (hacdias) [inline assets/src/i18n/index.js]: Sorry for being picky, I'll just ask to remove this, please

**Review by hacdias — CHANGES_REQUESTED** [review]
LGTM. Just remove that console.log and I'll merge this. And thank you 😄 a lot!
After merging this I'll take care of the "DRY" thing.

File: assets/src/i18n/index.js (hunk 1 above)
Comment (Equim-chan) [reply assets/src/i18n/index.js]: oh, my bad

**Comment by hacdias** [comment]
Thank you 😄

**Pull request merged**